tests/StripeChargeControllerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+
+namespace Tightenco\NovaStripe\Tests;
+
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Facades\Config;
+
+class StripeChargeControllerTest extends TestCase
+{
+    use WithFaker;
+
+    /** @test */
+    public function it_can_can_return_a_response()
+    {
+        $this->get('nova-vendor/nova-stripe/stripe/charges')
+            ->assertSuccessful();
+
+        $this->get('nova-vendor/nova-stripe/stripe/balance')
+            ->assertSuccessful();
+
+        $stripe = new \Stripe\StripeClient(Config::get('services.stripe.secret'));
+        $charge = $stripe->charges->create(
+            [
+            'amount' => $this->faker->numberBetween(50, 1000),
+            'currency' => 'usd',
+            'source' => 'tok_mastercard',
+            'description' => $this->faker->sentence,
+            ]
+        );
```

Comment:
```suggestion
        $charge = $stripe->charges->create([
            'amount' => $this->faker->numberBetween(50, 1000),
            'currency' => 'usd',
            'source' => 'tok_mastercard',
            'description' => $this->faker->sentence,
        ]);
```

Reply:
If we are creating real charges, we should probably make it clear that the stripe key and secret in `.env.testing` should be a test account